core/shared/src/main/scala/zio/ZIOAppArgs.scala

```diff
@@ -26,4 +26,10 @@ object ZIOAppArgs {
 
   def getArgs(implicit trace: ZTraceElement): ZIO[ZIOAppArgs, Nothing, Chunk[String]] =
     ZIO.serviceWith(_.getArgs)
+
+  /**
+   * Use when your App does not need to access the command-line arguments.
+   */
+  def empty(implicit trace: ZTraceElement): ULayer[ZIOAppArgs] = ZLayer.succeed(ZIOAppArgs(Chunk.empty))
```

Comment:
We should make this a value and provide an empty trace.

Reply:
Done, also used the new value in the test code, since it's been merged now